### PR TITLE
rgw: fix wrong debug related to user ACLs in rgw_build_bucket_policies()

### PR DESCRIPTION
**Pull request by rzarzynski**
Fixes: http://tracker.ceph.com/issues/19514
Signed-off-by: Radoslaw Zarzynski <rzarzynski@mirantis.com>

### CODE REVIEW

File: src/rgw/rgw_op.cc [hunk 1]
```diff
@@ -492,7 +492,7 @@ int rgw_build_bucket_policies(RGWRados* store, struct req_state* s)
     map<string, bufferlist> uattrs;
 
     ret = rgw_get_user_attrs_by_uid(store, acct_acl_user.uid, uattrs);
-    if (!ret) {
+    if (! ret) {
```
Comment (cbodley) [inline src/rgw/rgw_op.cc]: re: CodingStyle, the [google style guide](https://google.github.io/styleguide/cppguide.html) says:
```
// No spaces separating unary operators and their arguments.
x = -5;
++x;
if (x && !y)
```

Comment (rzarzynski) [inline src/rgw/rgw_op.cc]: Sure, fixing right now.

**Comment by rzarzynski** [comment]
jenkins test this please

**Comment by cbodley** [comment]
@rzarzynski rebase please

**Comment by cbodley** [comment]
@rzarzynski can you please rebase? the conflict looks relevent to this change in error handling

**Comment by yuriw** [comment]
@rzarzynski pls fix/rebase

--- pr 14369 --- pulling https://github.com/rzarzynski/ceph.git branch wip-rgw-19514
remote: Counting objects: 5, done.
remote: Total 5 (delta 3), reused 3 (delta 3), pack-reused 1
Unpacking objects: 100% (5/5), done.
From https://github.com/rzarzynski/ceph
 * branch            wip-rgw-19514 -> FETCH_HEAD
Auto-merging src/rgw/rgw_op.cc
CONFLICT (content): Merge conflict in src/rgw/rgw_op.cc
Adding qa/suites/upgrade/kraken-x/stress-split/objectstore/filestore-xfs.yaml
Adding qa/suites/upgrade/kraken-x/stress-split/objectstore/bluestore.yaml
Adding qa/suites/upgrade/kraken-x/stress-split/objectstore/.qa
Adding qa/suites/rgw/verify/frontend
Adding qa/suites/rbd/mirror/cluster
Adding qa/suites/rbd/mirror/base
Adding qa/suites/ceph-deploy/basic/distros/ubuntu_latest.yaml
Adding qa/suites/ceph-deploy/basic/distros/centos_latest.yaml
Adding qa/suites/ceph-deploy/basic/distros/.qa
Adding qa/suites/ceph-deploy/basic/config_options/.qa
Adding qa/suites/ceph-deploy/basic/ceph-deploy-overrides/.qa
warning: inexact rename detection was skipped due to too many files.
warning: you may want to set your merge.renamelimit variable to at least 2624 and retry the command.
Automatic merge failed; fix conflicts and then commit the result.
Traceback (most recent call last):
  File "/home/yuriw/wip_master/src/script/build-integration-branch", line 62, in <module>
    assert not r
AssertionError

**Comment by rzarzynski** [comment]
jenkins retest this please (docs failure because of network issues)

**Comment by yuriw** [comment]
[wip-yuri4-testing-2018-11-15-2120](https://trello.com/c/rED6z2kY/473-wip-yuri4-testing-2018-11-15-2120)

**Pull request merged**